src/App.tsx

```diff
@@ -1,14 +1,63 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [listOfTodos, setTodoList] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [serchQuery, setSearchQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const filterTodos = () => {
```

Comment:
better make it variable not a function. And add [useMemo](https://dmitripavlutin.com/react-usememo-hook/) for it and `filterTodos` become variable